client/src/components/PersonCard/personcard.test.js

```diff
@@ -1,16 +1,16 @@
 import React from 'react'
 import { shallow, configure } from 'enzyme'
-import Logo from './Logo.js'
+import PersonCard from './personcard'
 import Adapter from 'enzyme-adapter-react-16';
 configure({adapter: new Adapter()});
 
 // We will describe a block of tests
-describe('Logo component', () => {
+describe('PersonCard component', () => {
 	// we will write one individual test
   it('should render as expected', () => {
     // Shallow rendering renders a component without rendering any of its children
-    const component = shallow(<Logo />)
+    const component = shallow(<PersonCard />)
     // We create an assertion within the test that checks if our component renders our name prop
   // expect(component.contains('Your name')).toBe(true)
   })
-})
+})
```

Comment:
What happened here? Did you move the `Logo.test.js` file?

Reply:
I'm actually not sure, I don't remember what happened with this one. Is there something I should do to rectify or change the test.js?

Reply:
You can revert the single file!